pkg/resolver/search.go

```diff
@@ -395,11 +479,21 @@ func pointerToStringArray(pointerArray []*string) []string {
 	return values
 }
 
+func stringArrayToPointer(stringArray []string) []*string {
```

Comment:
Do we need this function? It looks like this it's only referenced from the test file.

<img width="721" alt="image" src="https://user-images.githubusercontent.com/4671325/168907513-79170c42-0708-4e3d-a3cb-659edb933bb3.png">

Reply:
Moved to test file